scholly/base/src/layouts/layout3/components/achievementsPage/Achievement.js

```diff
@@ -4,20 +4,31 @@
 
 import React from 'react';
 import { Card } from 'react-bootstrap';
-import { TEAM_ACHIEVEMENTS } from '../../../../global/data';
 import profilePicture from '../../../../shared/images/awardpic.jpg';
 
-const teamAchievements = TEAM_ACHIEVEMENTS;
+let bgTheme = 'bg-secondary';
+const Achievement = ({ achievement }) => {
+  bgTheme = (bgTheme === 'bg-secondary') ? 'bg-light' : 'bg-secondary';
 
-const Achievement = ({ achievement }) => (
-  <Card style={{ display: 'flex', flexDirection: teamAchievements.indexOf(achievement) % 2 === 0 ? 'row' : 'row-reverse' }} className="team-card">
-    <Card.Img style={{ width: '210px' }} variant="top" src={profilePicture} />
-    <Card.Body>
-      <div className="award-name">{achievement.title}</div>
-      <div className="award-position">{achievement.yearAwarded}</div>
-      <div className="award-summary">{achievement.description}</div>
-    </Card.Body>
-  </Card>
-);
+  return (
+
+    <div className="row mb-2" style={{ width: '70%' }}>
+      <Card className={`w-100 d-flex h-100 flex-column justify-content-center flex-wrap p-3 ${bgTheme}`}>
+        <div className="row g-0">
+          <div className="col-md-4 h-100">
+            <Card.Img variant="top" src={profilePicture} style={{ height: '150px', width: '150px', borderRadius: '100%' }} />
```

Comment:
Not sure why theres an image for the achievements page as currently we do not store any images for achievements and I don't think we intend on including this as a new feature. Might be good to remove this here rather than having to remove it after it gets merged into main.  In the case we do end up storing images for achievements then we could add this in later but not sure if we will end up doing it given the priorities with other tasks and the concern of images taking up a lot of storage on our database.

Reply:
Yep, noted - might be better to fix that up in a new PR though, just so we can have this one merged now, and focus on some of the other priorities.